app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -171,7 +182,11 @@ public function getStore($storeId = null)
     }
 
     /**
-     * {@inheritdoc}
+     * Retrieve stores array
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc